### DISCUSSION TITLE
Lesson 6: When I ctrl + "click vm" it doesn't jump.

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [Cyfrin/foundry-full-course-cu#270] [Q&A] [answered]
When I ctrl + " Click on vm " it doesn't jump, but the teacher in the video " Click on vm " will jump to the picture below!
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/f1efe5ad-8e69-4296-8d30-6b4082e371cf)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/07b68d33-f8e2-473e-aa70-8f405b9f6377)

**Comment by shikhar229169** [comment, marked as answer]
Yes, I was also facing this. So, I was checking whether ctrl click works on the Script, so it worked, and after doing that when I ctrl click on vm, it also worked. Try doing this.
So, what you have to do is, ctrl click the Script, and then try ctrl click on vm.
Edit: While importing you actually typed ```Scrip.sol```, it should be ```Script.sol```.

**Reply by IAm0xAxe** [reply]
Yeah,Thank you so much!
